### PR TITLE
resources: update "How to restore a wallet from mnemonic seed" user guide 

### PR DESCRIPTION
**Pull request by rating89us**

### CODE REVIEW

**Comment by rating89us** [comment]
Ready for review

File: _i18n/en/resources/user-guides/restore_account.md
Comment (erciccione) [inline _i18n/en/resources/user-guides/restore_account.md:21]: This page doesn't give information about restoring a hardware wallet. I would clarify or remove it altogether.

File: _i18n/en/resources/user-guides/restore_account.md
Comment (erciccione) [inline _i18n/en/resources/user-guides/restore_account.md:29]: This could be much shorter. IMO it's not necessary to add a dedicated explanation of the various type of seeds. You already mention above that there are more than one.

File: _i18n/en/resources/user-guides/restore_account.md
Comment (erciccione) [inline _i18n/en/resources/user-guides/restore_account.md:47]: I don't think we should add instructions for so many wallets. Only the GUI and CLI will do it. We don't need to provide instructions for every wallet we know. These guide will also need to be maintained.

File: _i18n/en/resources/user-guides/restore_account.md
Comment (erciccione) [inline _i18n/en/resources/user-guides/restore_account.md:87]:
```suggestion
9) On the next page, give your wallet a strong password and confirm it. Since your wallet will hold your private keys and seed, it is really important to enter a strong password.
```

It's not in the scope of this guide to give suggestion about password creation. Better leave that to the app themselves.

File: _i18n/en/resources/user-guides/restore_account.md
Comment (erciccione) [inline _i18n/en/resources/user-guides/restore_account.md:91]:
```suggestion
10) If your wallet is in `Advanced mode`, the `Daemon settings` page will be displayed. Here you can choose whether you want to start a local node or connect to a remote node. Choose your desired option and click on `Next` button.
```

Don't use `@` if the text is between backticks

**Review by erciccione — CHANGES_REQUESTED** [review]
I think this guide could be shorter. It would also be better if the original structure could be preserved. We have multiple translations for this guide and will be a waste to lose them because styiling changes.

Personally, where possible, i would prefer to keep the structure as it is (maybe keeping the additional 'When should I restore a wallet?') to avoid completely disregarding the translations we have, some of which were made by CCS funded contributors and funded by the community not long ago (for example German and Dutch).

Also, some sections that include more than 1 paragraph and images are duplicated for CLI and GUI. Would be better to have structure that include both, so that the text is not unnecessarely longer

File: _i18n/en/resources/user-guides/restore_account.md
Comment (rating89us) [reply _i18n/en/resources/user-guides/restore_account.md:21]: My original idea is to write a user guide on how to restore a wallet from a hardware wallet, but this guide doesn't exist yet. 

I could add a link to this non-existent page now and then we only merge this PR after the "how to restore a wallet from a hardware wallet" is created, or
I could remove it and then we add this link later, when the guide is created.

**Comment by rating89us** [comment]
> I think this guide could be shorter.

I agree.

> It would also be better if the original structure could be preserved. We have multiple translations for this guide and will be a waste to lose them because styiling changes.
> 
> Personally, where possible, i would prefer to keep the structure as it is (maybe keeping the additional 'When should I restore a wallet?') to avoid completely disregarding the translations we have, some of which were made by CCS funded contributors and funded by the community not long ago (for example German and Dutch).

Current guide is really low-quality and with outdated screenshots. I don't think it needs to be preserved.

> Also, some sections that include more than 1 paragraph and images are duplicated for CLI and GUI. Would be better to have structure that include both, so that the text is not unnecessarely longer

Created a dedicated section to explain restore height in details.

File: _i18n/en/resources/user-guides/restore_account.md
Comment (erciccione) [reply _i18n/en/resources/user-guides/restore_account.md]: We cannot merge it with a link to a document that doesn't exist yet. If that's the case, let's change this PR into a draft.

Comment (rating89us) [reply _i18n/en/resources/user-guides/restore_account.md]: Ok, I removed the link.

**Comment by rating89us** [comment]
Ready to be reviewed

**Comment by erciccione** [comment]
I forgot to mention before that you don't have to update every single translation manually. We have po4a which does that work for us (we will have to run it anyway). See https://github.com/monero-project/monero-site/blob/master/po/po-update.sh. The config file will be added after the changes to the english file are merged.

**Comment by rating89us** [comment]
Ok, I left only the English version changes.

**Review by erciccione — APPROVED** [review]

**Pull request merged**
